python-spec/src/somacore/data.py

```diff
@@ -205,7 +307,7 @@ def read(
         result_order: options.ResultOrderStr = _RO_AUTO,
         platform_config: Optional[options.PlatformConfig] = None,
     ) -> "SparseRead":
-        """Reads a subset of the object in one or more batches.
+        """Reads a subset this DenseNDArray in batches.
```

Comment:
Reads a subset _of_ ...

Reply:
Redid all of these to just say "the specified subarray" since you already know what type you're reading.